### PR TITLE
Run tests with Travis

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
[This Stack Overflow answer](http://stackoverflow.com/a/18457016) claims that this should work. We'll see.

File: tox.ini [hunk 1]
```diff
@@ -18,5 +18,5 @@ deps = flake8
 commands = flake8 distributionviewer
 
 [flake8]
-exclude = **/migrations/**,.tox
+exclude = **/migrations/**,.tox,distributionviewer/api/fakedata.py
```
Comment (robhudson) [inline tox.ini]: Or we could just fix the indenting?

**Comment by robhudson** [comment]
I think this looks good and Travis ran the tests. r+wc

**Comment by openjck** [comment]
Updated! I'll self-merge when the tests pass.

**Pull request merged**